Makefile

```diff
@@ -1,104 +1,38 @@
-.PHONY: rel deps test show_test_results
+.PHONY: rel
 LOG=$(subst TARGET,$@,TARGET.log 2>&1 || (cat TARGET.log; exit 1))
 SILENCE_COVER =  | grep -v "logs.*\\.coverdata"
 SILENCE_COVER += | grep -v "Analysis includes data from imported files"
 SILENCE_COVER += | grep -v "WARNING: Deleting data for module"
 LOG_SILENCE_COVER=$(subst TARGET,$@,TARGET.log 2>&1 || (cat TARGET.log $(SILENCE_COVER); exit 1))
 EJABBERD_DIR = apps/ejabberd
-EJD_INCLUDE = $(EJABBERD_DIR)/include
-EJD_PRIV = $(EJABBERD_DIR)/priv
 XEP_TOOL = tools/xep_tool
 EJD_EBIN = $(EJABBERD_DIR)/ebin
-DEVNODES = node1 node2 node3 fed1
-
-all: deps compile
-
-compile: rebar
-	./rebar $(OPTS) compile > $(LOG)
-
-deps: rebar
-	./rebar get-deps > $(LOG)
-
-clean: rebar configure.out
-	rm -rf apps/*/logs
-	. ./configure.out && ./rebar clean
-
-quick_compile: rebar
-	./rebar $(OPTS) compile skip_deps=true > $(LOG)
-
-reload: quick_compile
-	@E=`ls ./rel/mongooseim/lib/ | grep ejabberd-2 | sort -r | head -n 1` ;\
-	rsync -uW ./apps/ejabberd/ebin/*beam ./rel/mongooseim/lib/$$E/ebin/ ;\
-
-reload_dev: quick_compile
```

Comment:
I checked this and

> dev nodes use symlinks to deps

is true, but

> and MongooseIM's beam file[s]

is not:

```
17:14:16 erszcz @ x4 : /tmp/mongooseim (rebar3)
$ ls -l _build/mim1/lib/ | head -5
total 352
lrwxr-xr-x  1 erszcz  wheel    49B Dec 21 17:12 base16@ -> /private/tmp/mongooseim/_build/default/lib/base16
lrwxr-xr-x  1 erszcz  wheel    53B Dec 21 17:12 bbmustache@ -> /private/tmp/mongooseim/_build/default/lib/bbmustache
lrwxr-xr-x  1 erszcz  wheel    47B Dec 21 17:13 bear@ -> /private/tmp/mongooseim/_build/default/lib/bear
lrwxr-xr-x  1 erszcz  wheel    52B Dec 21 17:12 cache_tab@ -> /private/tmp/mongooseim/_build/default/lib/cache_tab
17:14:20 erszcz @ x4 : /tmp/mongooseim (rebar3)
$ ls -l _build/mim1/lib/ejabberd/ebin/ | head -5
total 9200
-rw-r--r--  1 erszcz  wheel    89K Dec 21 17:13 ELDAPv3.beam
-rw-r--r--  1 erszcz  wheel   6.9K Dec 21 17:13 XmppAddr.beam
-rw-r--r--  1 erszcz  wheel    12K Dec 21 17:13 acl.beam
-rw-r--r--  1 erszcz  wheel   9.0K Dec 21 17:13 adhoc.beam
```

I agree that just rebuilding the release is enough when we work on just one of them. I've been following this workflow for some time now and it's not bad, though it's a bit slower than just recompiling and copying the files over (especially for multiple release).

However, if we are to reintroduce this target, it'd be nice if it checked which releases are actually present in `_build` and copy the files to all of them instead of being hardcoded to just `mim1` / `mongoose_node1` or all of them and failing when one is not there.